front_end/html/edit_course.html

```diff
@@ -56,6 +56,32 @@ <h2>Create course</h2>
     <p><textarea name="passcode" class="textarea is-grey is-medium is-fullwidth monospace" rows="1">{% if course_details["passcode"] %}{{ course_details["passcode"] }}{% end %}</textarea></p>
 </div>
 
+<div class="shadow-box">
+    <div>
+        <p><strong>Enable research: </strong></p>
+            <div class="select is-grey">
+                <select name="enable_research" id="enable_research" class="edit-select" onchange="showConsentForm()">
+                {% if course_details["consent_text"] and course_details["consent_text"] != "" %}
+                    <option value="Yes" selected>Yes</option>
+                    <option value="No">No</option>
+                {% else %}
+                    <option value="Yes">Yes</option>
+                    <option value="No" selected>No</option>
+                {% end %}
+                </select>
+            </div>
+
+            <div id="consent_form_div" class="top-space" {% if course_details['consent_text'] is None or course_details['consent_text'] == '' %} style="display:none" {% end %}>
+                <p><strong>Research consent form*: </strong><br />Please use <a href="https://www.markdownguide.org/basic-syntax/" target='_blank' rel='noopener noreferrer'>Markdown syntax</a> to write a consent form for this exercise.</p>
+                <p><div class="textarea is-grey is-fullwidth monospace" data-placeholder="Please provide a consent form." style="min-height: 400px" id="consent_form" contenteditable="true">{% if course_details["consent_text"] %}{{ course_details["consent_text"] }}{% end %}</div></p>
+                <textarea name="consent_form_text" id="consent_form_text" class="invisible"></textarea>
+                <p class="top-space"><strong>Research consent alternative form*: </strong><br />Please use <a href="https://www.markdownguide.org/basic-syntax/" target='_blank' rel='noopener noreferrer'>Markdown syntax</a> to specify text that indicates what students should do if they do not consent to participate in the study.</p>
```

Comment:
Please change from "Research consent alternative form" to "Instructions for students who decline to participate in the study (must be specified if a consent form has been specified):"